adafruit_platformdetect/board.py

```diff
@@ -58,6 +58,7 @@
 ODROID_C1                   = "ODROID_C1"
 ODROID_C1_PLUS              = "ODROID_C1_PLUS"
 ODROID_C2                   = "ODROID_C2"
+ODROID_N2                   = "Hardkernel ODROID-N2"

```

Comment:
Just a little nitpick here, but maybe change the string to "ODROID_N2" so it's consistent with the other names.